requirements.txt

```diff
@@ -2,7 +2,7 @@ torch==1.10.*
 torchvision==0.11.*
 torchtext==0.11.*
 fairscale==0.4.*
-pytorch-lightning[extra]==1.5.*
+pytorch-lightning==1.5.*
```

Comment:
What is the reason for removing `[extra]`? It is needed for using `LightningCLI`.

Reply:
Maybe it is better to keep your change and add `jsonargparse[signatures]` (required by `LightningCLI`). This wouldn't pull other dependencies that are currently not used (like Horovod, ...).

Reply:
then I would consider:
a) add `jsonparser` directly
b) rase an issue on PL site to create another extra only for CLI

the reason I removed it is that there is some problem with building horovod which you do not need anyway...

Reply:
> This wouldn't pull other dependencies that are currently not used (like Horovod, ...).

exactly

Reply:
```suggestion
pytorch-lightning==1.5.*
jsonargparse[signatures]
```